docs/netlify/_redirects

```diff
@@ -0,0 +1,530 @@
+/docs/:major.:minor/frameworks-wrapper-for-react-redux-example.html /docs/:major.:minor/react-data-grid/redux/
+/docs/:major.:minor/frameworks-wrapper-for-react-simple-examples.html /docs/:major.:minor/react-data-grid/binding-to-data/
+/docs/:major.:minor/frameworks-wrapper-for-vue-custom-context-menu-example.html /docs/:major.:minor/javascript-data-grid/vue-custom-context-menu-example/
+/docs/:major.:minor/frameworks-wrapper-for-vue-custom-editor-example.html /docs/:major.:minor/javascript-data-grid/vue-custom-editor-example/
+/docs/:major.:minor/frameworks-wrapper-for-vue-custom-id-class-style.html /docs/:major.:minor/javascript-data-grid/vue-custom-id-class-style/
+/docs/:major.:minor/frameworks-wrapper-for-vue-custom-renderer-example.html /docs/:major.:minor/javascript-data-grid/vue-custom-renderer-example/
+/docs/:major.:minor/frameworks-wrapper-for-vue-hot-column.html /docs/:major.:minor/javascript-data-grid/vue-hot-column/
+/docs/:major.:minor/frameworks-wrapper-for-vue-hot-reference.html /docs/:major.:minor/javascript-data-grid/vue-hot-reference/
+/docs/:major.:minor/frameworks-wrapper-for-vue-installation.html /docs/:major.:minor/javascript-data-grid/vue-installation/
+/docs/:major.:minor/frameworks-wrapper-for-vue-language-change-example.html /docs/:major.:minor/javascript-data-grid/vue-language-change-example/
+/docs/:major.:minor/frameworks-wrapper-for-vue-simple-example.html /docs/:major.:minor/javascript-data-grid/vue-simple-example/
+/docs/:major.:minor/frameworks-wrapper-for-vue-vuex-example.html /docs/:major.:minor/javascript-data-grid/vue-vuex-example/
+ 
+/docs/:major.:minor/react-installation/ /docs/:major.:minor/react-data-grid/installation/
+/docs/:major.:minor/react-simple-example/ /docs/:major.:minor/react-data-grid/binding-to-data/
+/docs/:major.:minor/react-simple-examples/ /docs/:major.:minor/react-data-grid/binding-to-data/
+/docs/:major.:minor/react-modules/ /docs/:major.:minor/react-data-grid/modules/
+/docs/:major.:minor/react-hot-column/ /docs/:major.:minor/react-data-grid/hot-column/
+/docs/:major.:minor/react-setting-up-a-language/ /docs/:major.:minor/react-data-grid/language/
+/docs/:major.:minor/react-setting-up-a-locale/ /docs/:major.:minor/react-data-grid/numeric-cell-type/
+/docs/:major.:minor/react-custom-context-menu-example/ /docs/:major.:minor/react-data-grid/context-menu/
+/docs/:major.:minor/react-custom-editor-example/ /docs/:major.:minor/react-data-grid/cell-editor/
+/docs/:major.:minor/react-custom-renderer-example/ /docs/:major.:minor/react-data-grid/cell-renderer/
+/docs/:major.:minor/react-language-change-example/ /docs/:major.:minor/react-data-grid/language/
+/docs/:major.:minor/react-redux-example/ /docs/:major.:minor/react-data-grid/redux/
+/docs/:major.:minor/react-hot-reference/ /docs/:major.:minor/react-data-grid/methods/
+ 
+/docs/hyperformula https://hyperformula.handsontable.com/
+/docs/hyperformula/whatever https://hyperformula.handsontable.com/whatever
```

Comment:
Here should be implemented a dynamic redirect ("whatever" is an example phrase here) for example http://handsontable.com/docs/hyperformula/guide/advanced-usage.html -> https://hyperformula.handsontable.com/guide/advanced-usage.html.